lib/msal-common/test/cache/CacheManager.spec.ts

```diff
@@ -239,6 +245,29 @@ describe("CacheManager.ts test cases", () => {
         const filterTwo: AccountFilter = { homeAccountId: "Wrong Id" };
         accounts = cacheManager.getAccountsFilteredBy(filterTwo);
         expect(Object.keys(accounts).length).to.eql(0);
+
+        sinon.stub(TrustedAuthority, "getCloudDiscoveryMetadata").callsFake(() => {
```

Comment:
You can call `ClientTestUtils.setCloudDiscoveryMetadataStubs();` instead of setting this yourself both here and on L-289

Reply:
done.